cmd/config-sync-controllers/main.go

```diff
@@ -95,9 +95,10 @@ func main() {
 	})
 
 	syncPeriod := 10 * time.Minute
-	cacheBuilder := cache.MultiNamespacedCacheBuilder([]string{
-		*managedNamespace, controllers.OpenshiftConfigNamespace, controllers.OpenshiftManagedConfigNamespace,
-	})
+
+	opts := cache.Options{}
+	opts.Namespaces = []string{*managedNamespace, controllers.OpenshiftConfigNamespace, controllers.OpenshiftManagedConfigNamespace}
```

Comment:
You could inline all of this below on line 121, lets see if it works first
```
Cache: cache.Options{
  Namespaces: []string{...},
},
```

Reply:
Will do.